templates/new_device_login.txt

```diff
@@ -1,6 +1,8 @@
 {{t "New sign-in to Firefox" }}
 
 {{ device }}
+{{ ip }}
+{{ location }}
```

Comment:
@vladikoff Its already there in the `.html` partials for IP, we arent adding any extra device info here, so I guess that stays that way?